python/tvm/testing/utils.py

```diff
@@ -1953,10 +1953,13 @@ def expected(A: T.Buffer(1, "int32")):
     """
 
     def __init_subclass__(cls):
-        if hasattr(cls, "before"):
-            cls.before = cls._normalize_before(cls.before)
-        if hasattr(cls, "expected"):
-            cls.expected = cls._normalize_expected(cls.expected)
+        for name in ["before", "Before"]:
```

Comment:
Can we add an `assert` that at most one of the two options are present?

Reply:
I'm not sure if should be that strong of an assert, as that would prevent defining an intermediate test that defines `transform` and `expected`, but doesn't define `before`.  This could be useful for defining several tests that should all normalize to produce the same result.  This would also break existing cases where the `transform` is defined, then used across several tests in a file.

What if instead, we were to assert that no duplicates names exist?

```python
assert len([getattr(cls,name) for name in ['before','Before'] if hasattr(cls,name)]) <= 1
assert len([getattr(cls,name) for name in ['expected','Expected'] if hasattr(cls,name)]) <= 1
```